physics/module_MYNNPBL_wrapper.meta

```diff
@@ -327,17 +351,17 @@
   intent = out
   optional = F
 [hflx]
-  standard_name = kinematic_surface_upward_sensible_heat_flux_reduced_by_surface_roughness
-  long_name = kinematic surface upward sensible heat flux reduced by surface roughness
+  standard_name = kinematic_surface_upward_sensible_heat_flux
```

Comment:
Do you really need to change this? We discussed 2-3 weeks back that we should make MYNNPBL in principle compatible with the `lheatstrg` changes EMC made.

`kinematic_surface_upward_sensible_heat_flux_reduced_by_surface_roughness` is identical to `kinematic_surface_upward_sensible_heat_flux` as long as the canopy heat storage flag `lheatstrg` is set to `.false`.

Reply:
changed